target/linux/mediatek/image/Makefile

```diff
@@ -29,8 +29,8 @@ define Device/Default
   SUPPORTED_DEVICES := $(subst _,$(comma),$(1))
 ifeq ($(SUBTARGET),mt7623)
   KERNEL_NAME := zImage
-  KERNEL := kernel-bin | append-dtb | uImage none
-  KERNEL_INITRAMFS := kernel-bin | append-dtb | uImage none
+  KERNEL = kernel-bin | fit none $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
+  KERNEL_INITRAMFS :=
```

Comment:
If you are going to touch this, then why do you add the same lines in device recipes?
BTW why do you disable initramfs images? It can still be used for debugging purpose.

Reply:
It's only the legacy U7623 recipe which gets this. The others could even use `fit lzma` like everything else but there's no point in compression a since it's a zImage so I left it with "none".

Reply:
I put the initramfs image back.